sharktank/sharktank/types/tensors.py

```diff
@@ -1135,10 +1138,21 @@ def create(
     ) -> "InferenceTensor":
         shard_count = int(extra_properties["shard_count"])
         try:
-            ts = raw_tensors[""]
+            # We have to do this to avoid exporting as part of the `mlir` blob:
```

Comment:
Maybe this will fix what I have been observing that some constants appear in the MLIR.

Reply:
Yes. The copying semantic drops the externalization for flow tensor transfer. I think we should ultimately drop the transfer in the ReplicateTensor construction for a `torch.Tensor`. I honestly wish we could drop it altogether but materialized constants are going to be an issue we need to track down.